homepage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="styles.css" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Deuces</title>
+</head>
+<body>
+    <div>
+        <input type="search" name="search" value="search" placeholder="search">
+    </div>
+    <div class="category">
+            <textarea name="box1" id="" cols="30" rows="20">Box1</textarea>
+            <div class="advert1">
+                <textarea name="" id="" cols="30" rows="10"></textarea>
+            </div>
+    </div>
+    <div class="speak">
```

Comment:
Please use  descriptive class names. `speak` , `box1` do not describe anything

Reply:
Pls the right PR is my last commit others are at the beginning of learning